cmd/scip/test_test.go

```diff
@@ -0,0 +1,58 @@
+package main
+
+import (
+	"slices"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestTestCasesForLine(t *testing.T) {
```

Comment:
@matthewnitschke-wk Thank you for adding this test, it compactly expresses the different kinds of syntax that are available and how they get parsed. Nice!